project/SparkBuild.scala

```diff
@@ -671,7 +682,6 @@ object TestSettings {
       }
       Seq[File]()
     },
-    concurrentRestrictions in Global += Tags.limit(Tags.Test, 1),
```

Comment:
I think it should be okay to remove this here. The other subprojects still will not run in parallel because a) we use forking in tests and `testForkedParallel` will be false for those projects.